app/api/contests/contests.go

```diff
@@ -22,6 +23,21 @@ func (srv *Service) SetRoutes(router chi.Router) {
 
 	router.Put("/contests/{item_id}/additional-time", service.AppHandler(srv.setAdditionalTime).ServeHTTP)
 	router.Get("/contests/{item_id}/group-by-name", service.AppHandler(srv.getGroupByName).ServeHTTP)
+	router.Get("/contests/{item_id}/members-additional-times", service.AppHandler(srv.getMembersAdditionalTimes).ServeHTTP)
```

Comment:
For all these services, I wonder (for a while), if we shouldn't do `/contests/{item_id}/groups/{group_id}/members-additional-times" instead of the query parameter :-/
What do you think?

Reply:
The problem is that setAdditionalTime sets the additional time for the specified group while
getMembersAdditionalTimes returns additional times for sub-groups of the specified group. So GET & PUT for the same URL would work with different objects which is strange.

Reply:
The url can still show the difference with something like `GET /contests/:item_id/groups/:group_id/members/additional_time` vs `PUT /contests/{item_id}/groups/:group_id/additional-time` . 
May look complicated at first sight but make sense.

Reply:
done